app/app.rb

```diff
@@ -111,6 +118,7 @@ def add_rental
       date = gets.chomp
       rental_detail = Rental.new(date, @books[rental_book], @people[rental_person])
       @rentals.push(rental_detail)
+      save_rentals
       puts 'Rental Added Successfully'
     end
   end
```

Comment:
Great job team 🤝

- [ ] You have done a great job. However, your app breaks whenever I try to list the rentals. Kindly fix this error and resubmit for review.
![list-rentals](https://user-images.githubusercontent.com/63917725/225268588-60189b10-7102-4a76-825c-c6ebb61c04b7.PNG)